protocol/0091-LIQM-liquidity_mechanisms.md

```diff
@@ -0,0 +1,118 @@
+# On-Chain Liquidity Mechanisms
+
+## Summary
+
+The aim of the on-chain liquidity mechanisms are to reward parties for supplying competitively priced liquidity and facilitating the growth of a market. At a high level, the liquidity mechanisms are:
+
+- parties [accrue](#accruing-els-points) ELS points over time proportional to their maker fees earned.
+- parties are [designated](#designating-liquidity-providers) as LPs on an epoch by epoch basis if they receive above a specified proportion of the markets maker fees.
+- parties [receive](#distributing-liquidity-fees) a proportion of the liquidity fees accumulated in an epoch relative to their [accrued](#accruing-els-points) ELS points and [implied commitment amount](#implied-commitment-amount) (but only when designated as an LP).
+
+With the above mechanisms the protocol incentives the following desirable behaviour.
+
+- **Early adoption** - parties accrue ELS points which never decay over time. Early adopters of a market will benefit from accrued ELS points for the entire duration of the markets life-cycle.
+- **Tight spreads** - parties are incentivised to be at the front of the book as ELS points are only accrued when a party is the non-aggressive side of a trade.
+- **Deep liquidity** - parties are incentivised to supply a larger volume of notional liquidity within a configurable range to receive a larger share of the liquidity fees.
+- **Active market making** - parties who accumulate a large number of ELS points will still need to be designated as an LP in order to receive a proportion of the liquidity fees accumulated each epoch. Therefore parties are incentivised to continue to provide competitively priced liquidity even after accruing a large number of ELS points.
+
+The mechanisms also enable the following features:
+
+- **Key rotation** - ELS points are implemented as internal assets, as such they can be transferred between keys and traded.
+
+## Network parameters
+
+### `liquidityProvider.proportionRequirement`
+
+A number in the range $[0, 1]$ which defines the minimum proportion of a markets maker fees a party must receive in order to be [designated](#designating-liquidity-providers) as an LP for the next epoch. The parameter should default to `0.1`.
+
+Updates to this parameter will be used the next time LPs are designated, i.e. updating the network parameter will not result in LPs instantly being redesignated.
+
+## Accruing ELS Points
+
+Whenever a market proposal is enacted (passes opening auction), an internal Vega asset is created to track the ELS points of that market. For a successor market, a new asset should not be created and instead the ELS asset of the parent market should be used.
+
+At the end of each epoch, each party accrues ELS points for the relevant market as follows:
+
+$$ELS_{i_j} = V_{i_j} \cdot 10^{-Q}$$
```

Comment:
I was wondering if we wouldn't want to introduce some cut-off so that only some portion minimum proportion of maker fees in epochs qualifies for the ELS points, but the more I think about it the more I like the currently proposed design. It's in the spirit of user-ownership and could potentially incentivise smaller users to eventually become LPs or at least make some extra profit on the re-sale of their ELS points.

Reply:
Definitely agree with this point
> incentivise smaller users to eventually become LPs

On this point...

> make some extra profit on the re-sale of their ELS points

I think it's okay but something I was wondering is whether a late-arriving LPs with more capital would be able to buy up all the ELS points accrued by early traders wanting to dump? I think the early LPs should still have a much larger supply of ELS points in this case so we would be fine but worth considering.

Reply:
Not sure, though I think it's got more positive externalities. More fun to trade on markets and be earning something that's potentially worth something in the future and a late arrival of an LP commited enough to buy ELS off the smaller participants is probably a good thing for the market overall.